index.html

```diff
@@ -0,0 +1,205 @@
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+                <div class=" work wdr22">
+                    <h4>Profesional Art Printing Data More</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+                <div class=" work wdr23">
+                    <h4>Profesional Art Printing Data More</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+                <div class="work wdr31">
+                    <h4>Profesional Art Printing Data More</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+
+                <div class="work wdr32">
+                    <h4>Profesional Art Printing Data More</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+                <div class="work wdr33">
+                    <h4>Profesional Art Printing Data More</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+                </div>
+
+        </div>
```

Comment:
- You have done an amazing job here :muscle: but it is yet to be consistent with the project requirement.
- [ ] On each of your project cards, I suggest you check your `height`, `width`, `padding`, `margin`, `font-weight`, `alignment` of elements in their appropriate positions as seen in Fgma. Kindly check these images :point_down: 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/213870259-2e2c79d5-4d77-4ce4-8e51-41800fba03c9.png)|![image](https://user-images.githubusercontent.com/99715640/213870332-c06b135e-e69b-4c57-9a23-fd6235cfdac1.png)|